src/cmdlib.sh

```diff
@@ -285,6 +285,14 @@ runvm() {
 set -xeuo pipefail
 export PATH=/usr/sbin:$PATH
 workdir=${workdir}
+
+# use the builder user's id, otherwise some operations like
+# chmod will set ownership to root, not builder
+export USER=$(id -u)
+
+# ensure the user of files created do not have root ownership
+trap 'chown -h -R ${USER}:${USER} ${workdir}' EXIT
```

Comment:
not sure if this change will cause any problems or not, but I guess let's roll with them and find out.

Reply:
I saw problems the other way: permission denied when operating on the files created outside the container.